src/odemis/driver/test/tmcm-pd1240.tmcm.tsv

```diff
@@ -0,0 +1,12 @@
+# Parameters from PD-1240
```

Comment:
About the file extension, I'd suggest to use a different one than "tmcm.tsv", as it's a different format that these files. I had suggested "tmcc.tsv" in the spec (but feel free to pick something better).